js/gdm/authPrompt.js

```diff
@@ -7,9 +7,9 @@ const ByteArray = imports.byteArray;
 
 const Animation = imports.ui.animation;
 const Batch = imports.gdm.batch;
-const Config = imports.misc.config;
```

Comment:
It would be good to squash this commit with 3b0fb2c4f0f81b86fb2bd984fd951afdb11f506c (and perhaps a few bits of the previous commit as well)

Reply:
Done, squashed only this one and left the other as is.